deploy/template.yaml

```diff
@@ -854,6 +854,13 @@ Resources:
           PortMappings:
             - ContainerPort: 8080
               Protocol: tcp
+          HealthCheck:
+            Command:
+              - "CMD-SHELL"
+              - "curl -f <http://localhost:8080/healthcheck> || exit 1"
```

Comment:
I think those angle brackets are probably placeholders - you probably just want the URL?